src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+export function addBook(book) {
+  return {
+    type: 'CREATE_BOOK',
+    payload: book,
+  };
+}
```

Comment:
- Great job setting up your actions creators, however, you've not declared the variables for each action that are meant to follow the duck's pattern as this is a requirement. Please see the examples below to create them and modify your functions.

```
const CREATE_BOOK = 'bookstore/books/CREATE_BOOK';
```

Reply:
Thank you, sir, for raising my attention to these mistakes.